builder-base/Makefile

```diff
@@ -25,6 +31,12 @@ local-images:
 		--frontend dockerfile.v0 \
 		--opt platform=linux/amd64 \
 		--opt build-arg:BASE_IMAGE=$(BASE_IMAGE) \
+		--opt build-arg:BASE_IMAGE=$(BASE_IMAGE) \
```

Comment:
We are actually thinking of baking in multiple versions of go into one builder-base image for us to use for our different projects, which might result in us having to install them separately instead of passing them as parameters here. I guess we could also maintain different builder-base images based on the go version, but as it only acts as the image we use for our prow jobs and not images that we release as part of our kubernetes images, this might not bed needed.